pkg/flowmeter.go

```diff
@@ -4,8 +4,8 @@ import (
 	"flag"
 	"log"
 
-	"github.com/deepfence/FlowMeter/pkg/constants"
-	"github.com/deepfence/FlowMeter/pkg/packetAnalyzer"
+	"github.com/deepfence/FlowMeter-fix/pkg/constants"
+	"github.com/deepfence/FlowMeter-fix/pkg/packetAnalyzer"
```

Comment:
Ca you change it back to FlowMeter?

Reply:
Sure

Reply:
I've done the requested changes, please check and let me know